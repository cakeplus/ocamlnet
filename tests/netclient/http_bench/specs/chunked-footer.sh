. helpers.sh

start_test_server -line 1 -file data/chunked-footer
trap "stop_test_server" EXIT
request -get / -run
