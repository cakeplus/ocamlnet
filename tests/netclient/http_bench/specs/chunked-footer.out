884BB5731CDEA2C3DCE1E12F1C8C23AC
